winch/codegen/src/codegen/context.rs

```diff
@@ -501,6 +508,43 @@ impl<'a, 'builtins> CodeGenContext<'a, 'builtins> {
         }
     }
 
+    /// Truncates the value stack to the specified target, returning the amount
+    /// of memory entries truncated.
+    /// This function is intended to only be used when restoring the code
+    /// generation's reachability state, when handling an unreachable end or
+    /// else.
+    pub fn truncate_stack_to(&mut self, target: usize) {
```

Comment:
This is only called from `ensure_stack_state`, is it worth making it not `pub`?

Reply:
Fixed, thanks!